Dockerfile

```diff
@@ -16,12 +16,6 @@ curl libfontconfig nodejs npm nginx \
 
 RUN npm -g install grunt-cli bower
 
-# Set the locale
-RUN locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US:en
-ENV LC_ALL en_US.UTF-8
```

Comment:
can you paste here `locale` output from there then?

(we still could proceed with migrating to debian in order to shrink resulting image size)